plugins/backstage-plugin-devcontainers-react/src/hooks/useDevcontainers.ts

```diff
@@ -39,7 +39,8 @@ export function useDevcontainers(): UseDevcontainersResult {
   }
 
   const repoUrl = entity.metadata.annotations?.[ANNOTATION_SOURCE_LOCATION];
-  if (!repoUrl) {
+  const vsCodeUrl = entity.metadata.annotations?.vsCodeUri;
```

Comment:
Would be nice to export a constant for this, but then it would depend on the backend plugin and currently we say that the backend plugin is technically optional.  But, maybe we should make it a hard dependency.